Makefile

```diff
@@ -13,16 +13,13 @@ install:
 screenshots:
 	@rm -rf output && $(CYPRESS) run --browser chrome
 
-
 .PHONY: screenshots.spec
 screenshots.spec:
 	@rm -rf output && $(CYPRESS) run --browser chrome --spec=$(spec)
 
-
-.PHONY: screenshots.active-sessions
-screenshots.active-sessions:
-	@rm -rf output && $(CYPRESS) run --browser chrome --spec=cypress/e2e/active-sessions.cy.js
-
+.PHONY:screenshots-specific
```

Comment:
Can we delete .PHONY: screenshots.spec now?

Reply:
I guess we can. However, I'm not sure if there are any specific cases where you'd need to define the entire path of the test (such as in screenshots.spec), instead of just the test file name (as in screenshots-specification). Do you know?